interface/interface comic/update_web.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This script file should be moved to "ostis-art/scripts/" folder.
Only part with build of your component should be left. Please, rename the script as it should relate to component.
It'll be great to update "install_ostis.sh" to call this script so your web component will be ready on startup.